src/postings/skip.rs

```diff
@@ -172,12 +180,17 @@ impl SkipReader {
         let bytes = self.owned_read.as_slice();
         let advance_len: usize;
         self.last_doc_in_block = read_u32(bytes);
-        let doc_num_bits = bytes[4];
+        let mut doc_num_bits = bytes[4];
```

Comment:
`doc_num_bits` changes meaning here.
First it is something that encodes both `num_bits` and whether we need to do -1 stuff.
Then it is something that does encode only `num_bits`.

Can we make it two variables with different name?